pallets/rc-migrator/src/lib.rs

```diff
@@ -1295,11 +1307,120 @@ pub mod pallet {
 			);
 
 			Self::transition(MigrationStage::Pending);
-
 			Self::deposit_event(Event::MigrationCancelled);
 
 			Ok(Pays::No.into())
 		}
+
+		#[pallet::call_index(13)]
+		#[pallet::weight({ Weight::from_parts(10_000_000, 1000) })]
+		pub fn vote_manager_multisig(
+			origin: OriginFor<T>,
+			payload: Box<ManagerMultisigVote<T>>,
+			_sig: sp_core::sr25519::Signature,
+		) -> DispatchResult {
+			let _ = ensure_none(origin);
+
+			ensure!(ManagerMultisigRound::<T>::get() == payload.round, "RoundStale");
+			let mut votes_for_call = ManagerMultisigs::<T>::get(&payload.call);
+			ensure!(!votes_for_call.contains(&payload.who), "Duplicate");
+			votes_for_call.push(payload.who.clone());
+
+			if votes_for_call.len() >= T::MultisigThreshold::get() as usize {
+				let origin: <T as Config>::RuntimeOrigin =
+					frame_system::RawOrigin::Signed(Self::manager_multisig_id()).into();
+				let call = payload.call.clone();
+				let res = call.dispatch_bypass_filter(origin);
```

Comment:
Not sure if we actually need to bypass the filter? The RcMigrator pallet should not be disabled